packages/@react-spectrum/utils/src/Slots.tsx

```diff
@@ -57,14 +58,22 @@ export function SlotProvider(props) {
 
 export function ClearSlots(props) {
   let {children, ...otherProps} = props;
-  let content = children;
-  if (React.Children.toArray(children).length <= 1) {
-    if (typeof children === 'function') { // need to know if the node is a string or something else that react can render that doesn't get props
-      content = React.cloneElement(React.Children.only(children), otherProps);
+
+  const emptyObj = useMemo(() => ({}), []);
+
+  const content = useMemo(() => {
```

Comment:
it doesn't look like this memo is doing anything? was there a reason for doing this?

Reply:
Yeah you're right, I didn't notice that otherProps can never be stable. I've removed this bit from the PR